### DISCUSSION TITLE
Brickstore installation file recognised as Virus

### DISCUSSION THREAD

**Discussion by michiw101** [rgriebl/brickstore#713] [Q&A]
Hi there,

i have an issue starting or even downloading the installation file for Brickstore on Windows (x64).
After downloading, Windows 10 shows a message stating something like "something is wrong with the file" and a moment later Windows Defender shows a message that it deleted the file due to a virus found.

I tried downloading in Chrome, Edge and Firefox.

Any ideas whats wrong here? 
![error1](https://user-images.githubusercontent.com/130929789/232336814-08a9eca5-6f16-4f32-b8c4-48b0bc383659.PNG)
![error2](https://user-images.githubusercontent.com/130929789/232336833-2afbff98-b79b-4029-8a90-9f69c4b3c917.PNG)

**Comment by rgriebl** [comment]
Works perfectly fine for me (Windows 10 / x64 / Defender).
As always: all downloads from this GitHub repo are 100% clean. You can check the download URL on virustotal.com and you'll see that this is (once again) a false alarm.
https://www.virustotal.com/gui/file/341d690f12315ac5f7780703c4f158f08bf31755846e5147dcafbdfec18cba16/detection

There's nothing I can do about this from my side: you have to either update your defender signatures or simply white-list the BrickStore installer (check that file via virustotal.com before running the installation though to make sure that nothing on your PC or your network messed up the download).
